distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+
+[cols="4" options="header"]
+|===
+
+|File to Edit
+|Property(ies)
+|Original Value
+|Example of New Value
+
+|`bin/karaf.bat`
+|`address`
+|`5005`
+|`5006`
+
+.2+|`etc/org.apache.karaf.management.cfg`
+|`rmiRegistryPort`
+|`1099`
+|`1199`
+
+
+|`rmiServerPort`
+|`44444`
+|`44445`
+
+.2+|`etc/system.properties`
+|`httpsPort`,`port`
+|`8993`
+|`8994`
+
+|`httpPort`
+|`8181`
+|`8281`
+
+|===
+
+==== Editing ${branding} Web Service Providers Configuration Files
+
+[IMPORTANT]
+====
+If the hostname is changed during the install to something other than `localhost` a new keystore and truststore must be provided.
+====
+
+[TIP]
+====
+When changing the hostname for testing or development purposes, the installer can be started with a `?dev=true` URL query parameter. This causes the system to automatically generate self-signed certificates for any hostname that is entered during the install process.
+====
+
+==== Configuring Files in HOME Directory Hierarchy
+
+Many important configuration settings are stored in the `<${branding}_HOME>` directory.
+
+[IMPORTANT]
+====
+The passwords configured in this section reflect the passwords used to decrypt JKS (Java KeyStore) files.
+Changing these values without also changing the passwords of the JKS causes undesirable behavior.
+====
+
+. In `<${branding}_HOME>/etc/users.properties`, modify the line: `localhost=localhost,group,admin,manager,viewer,system-admin` to be: `<FQDN>=<PASSWORD>,group,admin,manager,viewer,system-admin`
+. Configure `<${branding}_HOME>/etc/system.properties`
+[source,bash]
+----
+#START ${branding} SETTINGS
+# Set the keystore and truststore Java properties
+javax.net.ssl.keyStore=etc/keystores/serverKeystore.jks
+javax.net.ssl.keyStorePassword=<NewPassword>
+javax.net.ssl.trustStore=etc/keystores/serverTruststore.jks
+javax.net.ssl.trustStorePassword=<NewPassword>
+javax.net.ssl.keyStoreType=jks
+
+# Set the global url properties
+org.codice.ddf.system.protocol=https://
+org.codice.ddf.system.hostname=<FQDN>
+org.codice.ddf.system.httpsPort=8993
+org.codice.ddf.system.httpPort=8181
+org.codice.ddf.system.port=8993
+org.codice.ddf.system.rootContext=/services
+
+# HTTPS Specific settings. If making a Secure Connection not leveraging the HTTPS Java libraries and
+# classes (e.g. if you are using secure sockets directly) then you will have to set this directly
+https.cipherSuites=TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_DSS_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA
+https.protocols=TLSv1.1,TLSv1.2
+----
+
+[NOTE]
+====
+Depending on the environment, it may be easier for integrators and administrators to configure ${branding} using the ${admin-console} prior to disabling it for hardening purposes.
+The ${admin-console} can be re-enabled for additional configuration changes.
+====
+
+In an environment hardened for security purposes, access to the ${ddf-branding} ${admin-console} might be denied.
```

Comment:
This paragraph should probably go somewhere close to the beginning of section 3.3 as it gives another reason for using config file configuration.

We may also want to reword it. Instead of having two separate sentences, we should mention that both, the admin console _and_ the "Karaf command line console" (which should be `${command-console}`) may not be available in a hardened environment and that using the later in such an environment may cause configuration errors.

Also, the second sentences contradicts the previous sentence saying "or the ${admin-console}", which isn't an option.